### PR TITLE
182210040 heroku deploy

### PR DESCRIPTION
**Pull request by ElissaDesign**
What does this PR do?
To make sure that the app can be deployed to heroku the Deployment Link is 
https://winners-c8-bn-be-staging.herokuapp.com/

### CODE REVIEW

**Review by T2003 — APPROVED** [review]

**Review by pacomesimon — APPROVED** [review]

**Review by nyakamwe — COMMENTED** [review]
There are conflicts with develop branch, so update your PR and ask to be merged to TTL

**Review by habaumugisha1 — COMMENTED** [review]
You don't have a Procfile in a root folder

**Comment by elissaihimbazwe** [comment]
> You don't have a Procfile in a root folder

what do you mean by a procfile ? I don't understand

File: Procfile [hunk 1]
```diff
@@ -0,0 +1 @@
+web:node src/index.js
```
Comment (habaumugisha1) [inline Procfile]: you can run the production script instead

Comment (habaumugisha1) [reply Procfile]: you can use web:npm start 

but you have to configure start script well

Comment (elissaihimbazwe) [reply Procfile]: Fixed it , you can review again

File: package.json [hunk 2]
```diff
@@ -6,14 +6,16 @@
     "build": "npx babel src --out-dir lib",
     "test": "nyc --reporter=text --reporter=html mocha --recursive --require @babel/polyfill --require @babel/register src/test",
     "coverage": "nyc  report --reporter=text-lcov --reporter=lcov",
-    "start": "node lib/index.js",
+    "start": "npx babel-node lib/index.js",
```
Comment (habaumugisha1) [inline package.json]: you don't need babel-node in production

Comment (habaumugisha1) [inline package.json]: and you need to add a migration script in start script

Comment (elissaihimbazwe) [reply package.json]: I finished them, you review them now

Comment (elissaihimbazwe) [reply package.json]: Removed it.

File: package.json [hunk 3]
```diff
@@ -6,14 +6,16 @@
     "build": "npx babel src --out-dir lib",
     "test": "nyc --reporter=text --reporter=html mocha --recursive --require @babel/polyfill --require @babel/register src/test",
     "coverage": "nyc  report --reporter=text-lcov --reporter=lcov",
-    "start": "node lib/index.js",
+    "start": "NODE_ENV=production npm run dbmigrate && node lib/index.js",
```
Comment (habaumugisha1) [inline package.json:9]: great work!

**Review by habaumugisha1 — APPROVED** [review]

File: package.json (hunk 3 above)
Comment (elissaihimbazwe) [reply package.json:9]: Thanks, but wait allitle, I'm still checking something

**Pull request merged**